### PR TITLE
mail-filter/postforward: Version bump to 1.1.1

### PR DESCRIPTION
**Pull request by Philippe23**
Just a copy of the 1.1.0 ebuild.

Closes: https://bugs.gentoo.org/687858
Closes: 12239
Signed-off-by: Philippe Chaintreuil <gentoo_bugs_peep@parallaxshift.com>
Package-Manager: Portage-2.3.66, Repoman-2.3.11

### CODE REVIEW

File: mail-filter/postforward/postforward-1.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline mail-filter/postforward/postforward-1.1.1.ebuild]: Get yourself a new calendar!

File: mail-filter/postforward/postforward-1.1.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (mgorny) [inline mail-filter/postforward/postforward-1.1.1.ebuild]: EAPI 7, please.

File: mail-filter/postforward/postforward-1.1.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	sed -i -e "s/@GENTOO_PORTAGE_EPREFIX@/${EPREFIX}/" *.go || die
+
+	# go build assumes files will be in src dir, but
+	# source files are in root in this package, so move
+	# them.
+	local new_src_dir="${S}/src/${EGO_PN}"
+	# Freak out if there's already something there because
+	# it means the package has changed and we'll need to
+	# adjust to it.
+	[[ ! -e "${new_src_dir}" ]] || die "${new_src_dir} already exists"
```
Comment (mgorny) [inline mail-filter/postforward/postforward-1.1.1.ebuild]: Don't you think double negation is confusing?

Comment (Philippe23) [reply mail-filter/postforward/postforward-1.1.1.ebuild]: I think I just cribbed this from another go ebuild, but changed to remove double-negative regardless.

I also updated the EAPI & Copyright and fixed the Github `Closes:`.

**Comment by Philippe23** [comment]
New commit message is now:

```
Copy of the 1.1.0 ebuild, update copyright & EAPI, and tweak to remove double-negative new_src_dir check.

Closes: https://bugs.gentoo.org/687858
Closes: https://github.com/gentoo/gentoo/pull/12239
Signed-off-by: Philippe Chaintreuil <gentoo_bugs_peep@parallaxshift.com>
Package-Manager: Portage-2.3.66, Repoman-2.3.11
```

**Comment by mgorny** [comment]
Hmm, I also see unnecessary eutils but I'll fix it while merging. Thanks!

**Pull request closed without merging**